Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -136,9 +136,15 @@ class TabCell: UICollectionViewCell {
         self.title = title
     }
 
-    func setTabSelected(_ isPrivate: Bool) {
-        // This creates a border around a tabcell. Using the shadow craetes a border _outside_ of the tab frame.
-        layer.shadowColor = (isPrivate ? UIConstants.PrivateModePurple : UIConstants.SystemBlueColor).cgColor
+    func setTabSelected(forType type: TabType) {
+        // This creates a border around a tabcell. Using the shadow creates a border _outside_ of the tab frame.
+        switch type {
```

Comment:
I see both approaches for utilizing themes, some are just using ternary and switch being used here for example. Maybe we should centralize or is this a slowly progressing thing?

Reply:
I did not refactor as themes and tabs should be refactored at some point however the risk was low so I have refactored to use `switch` statements